firefox-ios/firefox-ios-tests/Tests/XCUITests/TodayWidgetTests.swift

```diff
@@ -0,0 +1,443 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import XCTest
+
+// Selectors
+
+// Widget buttons
+let editWidgetButton = "com.apple.springboardhome.application-shortcut-item.configure-widget"
+let editHomeScreenButton = "com.apple.springboardhome.application-shortcut-item.rearrange-icons"
+let removeWidgetButton = "com.apple.springboardhome.application-shortcut-item.remove-widget"
+
+// Buttons Identifier
```

Comment:
For the last 2 we already have identifiers added inside AccessibilityIdentifiers struct. You should use them and add one for the first button.